web/styles/inputs.css

```diff
@@ -0,0 +1,95 @@
+.input-active-styles {
+    color: var(--color-text-input-focus);
+    background-color: var(--color-background-input-focus);
+    border-color: var(--color-border-input-focus);
+}
+
+.input-element-wrapper {
+    display: grid;
+    grid-template:
+        [input-element-start] "icon-left-margin input-icon icon-content-gap content button-content-gap input-button button-right-margin" auto [input-element-end] / [input-element-start] var(
+            --input-icon-left-margin
```

Comment:
Rather than the somewhat misleading `-left-margin` and `-right-margin` CSS variable names and grid areas, I've been using `-offset` for such things. That gives a little flexibility for where those spatial values can be used, and avoids incorrectly suggesting that they'll appear with `margin` properties...

Reply:
I have renamed the variables to "icon-starting-offset" and "button-ending-offset" to make the naming more consistent with those used beforehand and for clarity.

Reply:
Perfect.